src/TestRunner-Core/TestRunner2.class.st

```diff
@@ -0,0 +1,330 @@
+"
+I am a more advenced TestRunner. I can run several tests (from a collection, from a giving package). I can accumulate or not tests result when executing test class separatly.
+
+# Public API
+- run:
+- runAll:
+- runPackage:
+- runPackages:
+- switchToDebugMode
+- switchToRunMode
+- accumulateResult
+- doNotAccumulateResult
+- runFailures
+"
+Class {
+	#name : #TestRunner2,
```

Comment:
Why not call it AdvancedTestRunner then (following what the comment says)?

Reply:
Yess i think it is a good idea to change the name.